blockchain/blockchain-wallet-v4-frontend
# fix(bank linking): adding more bank icons and possible bank names and…

## Description (optional)
https://blockchain.atlassian.net/browse/FWLT-998

## Testing Steps (optional)
add a test bank account:
```
Provider Name | User Login Name | Password
Dag Site | YodTest.site16441.2 | site16441.2
```
and make sure it looks correct in simple buy form, simple buy payment methods section, and user settings.
